Formula/redis@1.3.rb

```diff
@@ -0,0 +1,65 @@
+class RedisAT13 < Formula
+  desc "Persistent key-value database, with built-in net interface"
+  homepage "https://redis.io/"
+  url "https://github.com/antirez/redis/archive/v1.3.12.tar.gz"
+  sha256 "326d8866317f7bcaafba80409b847700620025dbd34e638f85f05bc1c13beabd"
+  head "https://github.com/antirez/redis.git", :branch => "unstable"
+
+  keg_only :versioned_formula
+
+  def install
+    # Architecture isn't detected correctly on 32bit Snow Leopard without help
+    ENV["OBJARCH"] = MacOS.prefer_64_bit? ? "-arch x86_64" : "-arch i386"
+
+    # Head and stable have different code layouts
+    src = (buildpath/"src/Makefile").exist? ? buildpath/"src" : buildpath
+    system "make", "-C", src, "CC=#{ENV.cc}"
+
+    %w[redis-benchmark redis-cli redis-server redis-check-dump redis-check-aof].each do |p|
+      begin
+        bin.install "#{src}/#{p}"
+      rescue
+        nil
+      end
+    end
+
+    %w[run db/redis log].each { |p| (var+p).mkpath }
+
+    # Fix up default conf file to match our paths
+    inreplace "redis.conf" do |s|
+      s.gsub! "/var/run/redis.pid", "#{var}/run/redis.pid"
+      s.gsub! "dir ./", "dir #{var}/db/redis/"
+    end
+
+    etc.install "redis.conf" unless (etc/"redis.conf").exist?
+  end
+
+  plist_options :manual => "redis-server #{HOMEBREW_PREFIX}/etc/redis.conf"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>KeepAlive</key>
+        <true/>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_prefix}/bin/redis-server</string>
+          <string>#{etc}/redis.conf</string>
+        </array>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>WorkingDirectory</key>
+        <string>#{var}</string>
+        <key>StandardErrorPath</key>
+        <string>#{var}/log/redis.log</string>
+        <key>StandardOutPath</key>
+        <string>#{var}/log/redis.log</string>
+      </dict>
+    </plist>
+        EOS
+  end
```

Comment:
Thanks, Mike! This formula adds a `redis-server` instead of `redis28` like prefix. So, I have added `redis-server`.

I didn't want to change how these were added previously in the https://github.com/Homebrew/homebrew-versions/pull/1537. If `redisXX-server` is preferred, let me know. I'll update all the files.